databases/week1/queries.sql

```diff
@@ -0,0 +1,37 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(*) FROM task;
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT * FROM task WHERE due_date IS NULL;
```

Comment:
this query is wrong, please take a look

Reply:
Hi jpganz, 
This query has been improved now. Thank you for your review!